src/main.py

```diff
@@ -48,7 +54,13 @@ def dataset_by_id(request: Request, response: Response, id: str):
     else:
         response.status_code = status.HTTP_406_NOT_ACCEPTABLE
         return
-    
+
+@app.get("/datasets/{dataset_id}/editions/{edition_id}/versions/{version_id}")
+def version(dataset_id: str, edition_id: str, version_id: str):
```

Comment:
can you add the default `request: Request, response: Response` args please. We dont need them yet but we will need them, best to keep the pattern going.